pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -40405,7 +40405,6 @@ self: {
        description = "Binary and exponential searches";
        license = stdenv.lib.licenses.bsd3;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
This file shouldn't be edited directly, but instead the `- binary-search` line should be removed from the `broken-packages` section of `pkgs/development/haskell-modules/configuration-hackage2nix.nix`.